charms/kubernetes_snaps.py

```diff
@@ -417,6 +417,8 @@ def configure_scheduler(extra_args_config, kubeconfig):
 
     scheduler_opts["v"] = "2"
     scheduler_opts["config"] = kube_scheduler_config_path
+    scheduler_opts["authorization-kubeconfig"] = kubeconfig
+    scheduler_opts["authentication-kubeconfig"] = kubeconfig
```

Comment:
wow, these are separate from the `scheduler_config.clientConnection`?

Got a link to the docs on that one?

Reply:
This PR provides a summary: https://github.com/kubernetes/kubeadm/issues/1285
We also use delegation for auth checks in the `kube-controller-manager` configuration.